src/commands/asset-compute/devtool.js

```diff
@@ -12,20 +12,30 @@
 
 'use strict';
 
-const devtool = require('@adobe/asset-compute-devtool');
-
+const { DevtoolServer } = require('@adobe/asset-compute-devtool');
+const { createHttpTerminator } = require('http-terminator');
 const BaseCommand = require('../../base-command');
 const { flags } = require('@oclif/command');
+const util = require('../../lib/util');
```

Comment:
Don't use `util.log`. It's the same as `console.log()`: https://github.com/adobe/aio-cli-plugin-asset-compute/blob/master/src/lib/util.js#L35

Most of the `util.js` and its logging stuff only exists because it's legacy code that was copied over from the nui cli and I didn't want to rewrite everything back then.